web-ui/src/main/resources/catalog/components/common/ows/OWSService.js

```diff
@@ -116,14 +184,16 @@
                 getFlatLayers(layer[i], inheritedCrs);
               }
             } else if (angular.isDefined(layer)) {
+
               // replace with complete CRS list if available
               if (layer.CRS && layer.CRS.length > inheritedCrs.length) {
                 inheritedCrs = layer.CRS;
               }
 
-              // add to flat layer array if we're on a leave (layer w/o child)
+              // add to flat layer array if we're on a leaf (layer w/o child)
               layer.url = url;
               layer.CRS = inheritedCrs;
+              layer.EX_GeographicBoundingBox = layer.EX_GeographicBoundingBox || bbox;
```

Comment:
Also inherit the geographic bounding box if the "child" does not have that property.